<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>32</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>32</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>32</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>32</number>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>6</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>7</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>8</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>9</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>10</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>11</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>12</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>13</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>14</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>15</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_pow3">
            <property name="text">
             <string>Найти элементы степени 3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Количество:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_sumMod3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_pow2">
            <property name="text">
             <string>Найти элементы степень 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_sumMod2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Количество:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="plainText">
           <string>Нажми клавишу &lt;&lt;2&gt;&gt;,
чтобы найти элементы,
которые являются степенью 2
   1 = 2^0     64 = 2^6
   2 = 2^1    128 = 2^7
   4 = 2^2    256 = 2^8
   8 = 2^3    512 = 2^9
  16 = 2^4   1024 = 2^10
  32 = 2^5   2048 = 2^11

Нажми клавишу &lt;&lt;3&gt;&gt;,
чтобы найти элементы,
которые являются степенью 3
   1 = 3^0    729 = 3^6
   3 = 3^1   2187 = 3^7
   9 = 3^2   6561 = 3^8
  27 = 3^3  19683 = 3^9
  81 = 3^4  59049 = 3^10
 243 = 3^5 117147 = 3^11

Нажми клавишу &lt;&lt;G&gt;&gt;,
чтобы сгенерировать новую таблицу</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Чётные числа</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Нечётные числа</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_generateTable">
        <property name="text">
         <string>Генерировать таблицу</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
